.python-version

```diff
@@ -0,0 +1 @@
+2.7.8
```

Comment:
This thingy appeared again. :)

(Where does it come from?)

If it's semi-common, you could add it to our .gitignore file.

Reply:
arr >:(
I'm sorry. I usually add it. Thanks Mike